src/eos_gates_windows.js

```diff
@@ -39,14 +39,17 @@ function loadJSON(path) {
 
 function readWhitelist() {
     let dataDirs = GLib.get_system_data_dirs();
-    dataDirs.forEach(function(dataDir) {
+    return dataDirs.map((dataDir) => {
```

Comment:
This whole whitelist thing seems a little pointless given we don't ship Wine. Should we just remove it and re-instate the functionality later/never? If we _did_ want to ship a Windows app run against Wine, we'd probably do it in an extra_data Flatpak linked to a Wine runtime / helper app, so that it could be tested/released as a working unit. So we could just reference that Flatpak from whatever source.